generate/lora.py

```diff
@@ -111,6 +111,7 @@ def main(
     fabric.print(f"Time to load the model weights: {time.time() - t0:.02f} seconds.", file=sys.stderr)
 
     model.eval()
+    merge_lora_weights(model)
```

Comment:
Is this in case the weights haven't been merged already?

What's more common, checkpoints with the weights merged or unmerged?

Reply:
Yeah, the checkpoint produced by the lora finetuning scripts only saves the lora weights, meaning they are not merged. This is common, because we can store these in a few MB vs. a full checkpoint (GBs). Additionally, it is common to share the LoRA weights if the original weights had a license that prevented the distribution of checkpoints (LLaMA v1). Plus, the merging is very cheap. So there are only benefits to this.

Reply:
Oh and you see in the line above `model.eval()`, this is where the merging happened previously, so we're not changing  how it was before.